image-search/Models/Album.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Album.swift
+//  image-search
+//
+//  Created by Renato Vieira on 08/10/19.
+//  Copyright © 2019 Stant. All rights reserved.
+//
+
+import Foundation
+
```

Comment:
Why is called search images? 
It is better something related to album or record.

Reply:
Changed to album.